src/cloudevent/partials/partials.ts

```diff
@@ -1,31 +1,27 @@
-import {alertsOnAlertPublished} from './alerts/alerts-on-alert-published';
-import {alertsCrashlyticsOnNewAnrIssuePublished} from './alerts/crashlytics-on-new-anr-issue-published';
-import {alertsCrashlyticsOnNewFatalIssuePublished} from './alerts/crashlytics-on-new-fatal-issue-published';
-import {alertsCrashlyticsOnNewNonfatalIssuePublished} from './alerts/crashlytics-on-new-nonfatal-issue-published';
-import {alertsCrashlyticsOnRegressionAlertPublished} from './alerts/crashlytics-on-regression-alert-published';
-import {alertsCrashlyticsOnStabilityDigestPublished} from './alerts/crashlytics-on-stability-digest-published';
-import {alertsCrashlyticsOnVelocityAlertPublished} from './alerts/crashlytics-on-velocity-alert-published';
-import {
-  alertsAppDistributionOnNewTesterIosDevicePublished
-} from './alerts/app-distribution-on-new-tester-ios-device-published';
-import {alertsBillingOnPlanAutomatedUpdatePublished} from './alerts/billing-on-plan-automated-update-published';
-import {alertsBillingOnPlanUpdatePublished} from './alerts/billing-on-plan-update-published';
-import {eventarcOnCustomEventPublished} from './eventarc/eventarc-on-custom-event-published';
-import {storageOnObjectArchived} from './storage/storage-on-object-archived';
-import {storageOnObjectDeleted} from './storage/storage-on-object-deleted';
-import {storageOnObjectFinalized} from './storage/storage-on-object-finalized';
-import {storageOnObjectMetadataUpdated} from './storage/storage-on-object-metadata-updated';
-import {pubsubOnMessagePublished} from './pubsub/pubsub-on-message-published';
+import { MockCloudEventPartials } from '../types';
+import { alertsOnAlertPublished } from './alerts/alerts-on-alert-published';
+import { alertsCrashlyticsOnNewAnrIssuePublished } from './alerts/crashlytics-on-new-anr-issue-published';
+import { alertsCrashlyticsOnNewFatalIssuePublished } from './alerts/crashlytics-on-new-fatal-issue-published';
+import { alertsCrashlyticsOnNewNonfatalIssuePublished } from './alerts/crashlytics-on-new-nonfatal-issue-published';
+import { alertsCrashlyticsOnRegressionAlertPublished } from './alerts/crashlytics-on-regression-alert-published';
+import { alertsCrashlyticsOnStabilityDigestPublished } from './alerts/crashlytics-on-stability-digest-published';
+import { alertsCrashlyticsOnVelocityAlertPublished } from './alerts/crashlytics-on-velocity-alert-published';
+import { alertsAppDistributionOnNewTesterIosDevicePublished } from './alerts/app-distribution-on-new-tester-ios-device-published';
+import { alertsBillingOnPlanAutomatedUpdatePublished } from './alerts/billing-on-plan-automated-update-published';
+import { alertsBillingOnPlanUpdatePublished } from './alerts/billing-on-plan-update-published';
+import { eventarcOnCustomEventPublished } from './eventarc/eventarc-on-custom-event-published';
+import { pubsubOnMessagePublished } from './pubsub/pubsub-on-message-published';
+import { storageV1 } from './storage';
```

Comment:
Shouldn't there be a new file here?

Reply:
src/cloudevent/partials/partials.ts:14:27 - error TS2307: Cannot find module './storage' or its corresponding type declarations.

14 import { storageV1 } from './storage';

Reply:
Indeed. Added missing file back.